### ISSUE TITLE
Patch file missing information

### ISSUE THREAD

**Issue by ArtemGordon** [insidesherpa/bank-mergeCoTechTask1#45]
Hello, my patch file is missing some information. Most of the getDataPoint function is not showing specifically where I coded to get the average price. It then jumps to the getRatio function. When opening the patch file on console it shows missing information about the getDataPoint when I have actually completed it and ran the task with no errors.  There is also a section of unknown illegible text in the patch file. I have attached images. 

Thanks!

![Screen Shot 2020-04-12 at 4 13 25 PM](https://user-images.githubusercontent.com/63560871/79078836-365c7d80-7cd9-11ea-88f6-e27d5255b170.png)

![Screen Shot 2020-04-12 at 4 13 30 PM](https://user-images.githubusercontent.com/63560871/79078837-36f51400-7cd9-11ea-8b6e-08b1be34a95b.png)

![Screen Shot 2020-04-12 at 4 13 38 PM](https://user-images.githubusercontent.com/63560871/79078838-36f51400-7cd9-11ea-9e1e-d5c6557e1683.png)

![Screen Shot 2020-04-12 at 4 13 48 PM](https://user-images.githubusercontent.com/63560871/79078839-36f51400-7cd9-11ea-9f2d-5a196c03e22f.png)

![Screen Shot 2020-04-12 at 4 14 23 PM](https://user-images.githubusercontent.com/63560871/79078840-378daa80-7cd9-11ea-89e8-3c7857e9a70f.png)

![Screen Shot 2020-04-12 at 4 14 55 PM](https://user-images.githubusercontent.com/63560871/79078841-378daa80-7cd9-11ea-86c0-4a7748e5f9bc.png)

![Screen Shot 2020-04-12 at 4 20 58 PM](https://user-images.githubusercontent.com/63560871/79078919-aa972100-7cd9-11ea-9c02-a55aecd7a890.png)

**Comment by yadvendra20** [comment]
Yes, I am also facing the same problem

**Comment by rohitnair566** [comment]
Yes, i am also facing the  same problem ,have you resolved your problem yet

**Comment by ArtemGordon** [comment]
Yes and no. I am not sure what the cause of the issue was but I rewrote the
entire assignment and it came out correct this time.
